plugins/repository-s3/src/main/java/org/opensearch/repositories/s3/S3BlobContainer.java

```diff
@@ -296,6 +297,35 @@ private static DeleteObjectsRequest bulkDelete(String bucket, List<String> blobs
             .build();
     }
 
+    @Override
+    public void listBlobsByPrefixInSortedOrder(
```

Comment:
Changes in this file will be added as part of https://github.com/opensearch-project/OpenSearch/pull/8314. Reviewers can ignore these changes in this PR.